packages/plugin-electron-client-state-persistence/src/crash_handler-posix.c

```diff
@@ -0,0 +1,46 @@
+#include "crash_handler.h"
+
+#include <signal.h>
+#include <stdlib.h>
+
+static const int bsg_native_signals[] = {SIGILL, SIGTRAP, SIGABRT,
+                                         SIGBUS, SIGFPE,  SIGSEGV};
+#define SIGNAL_COUNT 6
```

Comment:
This made me realize I can probably do one better - since this code no longer has to pass through the MS compiler, the count can be generated from the list itself, removing the need for the constant. :crossed_fingers: